modules/ROOT/partials/misc/plugin-toolbar-button-id-boilerplate.adoc

```diff
@@ -1,28 +1,29 @@
-{% if page.name == "available-toolbar-buttons.md" %}
+ifeval::["{docname}" == "advanced/available-toolbar-buttons"]
```

Comment:
We really need to get those first 2 PRs merged as Lorenzo also had to fix these pages... I had mentioned it this morning in hopes to get the approvals needed, but I'll try again tomorrow sorry.

Reply:
After rebasing off of the latest release/TinyMCE_5.10 things look good to me